### DISCUSSION TITLE
"Other" meaning

### DISCUSSION THREAD

**Discussion by knpshkn** [akvorado/akvorado#264] [Q&A] [answered]
Could you please explain what "other" means on visualize page and which flows fall under it. I understand that this is some kind of aggregation, but I can't understand the criteria for this. Maybe these are records from consolidated tables?

**Comment by vincentbernat** [comment, marked as answer]
Flows are grouped by the dimensions you specify. Only the N top groups are shown, the others are put in "Other". N is the limit defined below the dimensions.

**Reply by Knot3n** [reply]
Thanks for the explanation, we thought that "Other" is traffic that does not include any DST AS. (not known)

Perhaps, a clarification here would be easier for all other users, before the next one comes and asks this question.

**Reply by knpshkn** [reply]
for example, in this case, I do not really understand the meaning of "others"
<details>
  <summary>screenshot</summary>
  

![akvorado_other](https://user-images.githubusercontent.com/49982467/208443321-a7f93e39-ddad-47d6-8a95-ce4ececf2dc3.png)
</details>

**Reply by vincentbernat** [reply]
On the third column, "Others" mean "neither amazon, neither s3, neither ec2". There is a finite number of things that can be displayed on each column, the remaining ones are aggregated into "Others".

**Reply by lephuhai270** [reply]
Could you please advise how can we know those things in "Other"?

**Reply by vincentbernat** [reply]
Increase the limit.

**Reply by lephuhai270** [reply]
Thanks for your quick answer
